Resources/views/line/folder.html.twig

```diff
@@ -0,0 +1,7 @@
+{% if subLinks|length > 0 %}
+    <ul>
```

Comment:
You could create some folder-view-specific CSS classes to make CSS styling easier.